packages/fuels-core/src/constants.rs

```diff
@@ -4,6 +4,7 @@ use fuel_types::AssetId;
 pub const DEFAULT_GAS_LIMIT: u64 = 1_000_000;
 pub const DEFAULT_GAS_PRICE: u64 = 0;
 pub const DEFAULT_MATURITY: u64 = 0;
+pub const DEFAULT_SPEND_MESSAGE: bool = false;
```

Comment:
Is this used anywhere? It looks like everywhere that builds a `Message` struct is actually defaulting `fuel_block_spend` to `None`